docs/source/usersguide/mgxs_library.rst

```diff
@@ -0,0 +1,289 @@
+
+  :absorption:
+    This element requires the group-wise absorption cross section ordered by
+    increasing group index (i.e., fast to thermal).  If ``representation`` is
+    ``isotropic``, then the length of this list should equal the number of
+    groups described in the ``groups`` element.  If ``representation`` is
+    ``angle``, then the length of this list should equal the number of groups
+    times the number of azimuthal angles times the number of polar angles,
+    with the inner-dimension being groups, intermediate-dimension being
+    azimuthal angles and outer-dimension being the polar angles.
+
+    *Default*: None, this must be provided.
+
+  :scatter:
+    This element requires the scattering moment matrices presented with the
+    columns representing incoming group and rows representing the outgoing
+    group.  That is, down-scatter will be above the diagonal of the resultant
+    matrix.  This matrix is repeated for every Legendre order (in order of
+    increasing orders) if ``scatt_type`` is ``legendre``; otherwise, this
+    matrix is repeated for every bin of the histogram or tabular
+    representation.  Finally, if ``representation`` is ``angle``, the above
+    is repeated for every azimuthal angle and every polar angle, in that
+    order.
+
+    *Default*: None, this must be provided.
+
+  :multiplicity:
+    This element provides the ratio of neutrons produced in scattering
+    collisions to the neutrons which undergo scattering collisions; that is,
+    the multiplicity provides the code with a scaling factor to account for
+    neutrons being produced in (n,xn) reactions.  This information is assumed
+    isotropic and therefore does not need to be repeated for every Legendre
+    moment or histogram/tabular bin.  This matrix follows the same arrangement
+    as described for the ``scatter`` element, with the exception of the
+    data needed to provide the scattering type information.
+
+    *Default*: Multiplicities of 1.0 are assumed (i.e., (n,xn) reactions are
+    neglected).
+
+  The following fission-specific data are only needed should ``fissionable``
+  be ``true``.
+
+  :fission:
+    This element requires the group-wise fission cross section ordered by
+    increasing group index (i.e., fast to thermal).  If ``representation`` is
+    ``isotropic``, then the length of this list should equal the number of
+    groups described in the ``groups`` element.  If ``representation`` is
+    ``angle``, then the length of this list should equal the number of groups
+    times the number of azimuthal angles times the number of polar angles,
+    with the inner-dimension being groups, intermediate-dimension being
+    azimuthal angles and outer-dimension being the polar angles.
+
+    *Default*: None, this is required only if ``fission`` tallies are
+    requested and the material is fissionable.
+
+  :k_fission:
+    This element requires the group-wise kappa-fission cross section ordered by
```

Comment:
Maybe have a hyperlink to the description of the kappa-fission score. Also consider naming it `kappa_fission` instead of `k_fission`, which makes me think eigenvalue.